example/demo.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+var Stubby = require('stubby');
```

Comment:
Think this wouldn't work. Think for the demo, `Stubby` is added to the window before this `js` file is loaded.